src/services/Tex2Svg.php

```diff
@@ -119,7 +120,15 @@ private function runNodeApp(string $content): string
         unlink($tmpFile);
 
         if (!$process->isSuccessful()) {
-            throw new ProcessFailedException('PDF generation failed during Tex rendering.', 0, new SymfonyProcessFailedException($process));
+            $log = (new Logger('tex2svg'))->pushHandler(new ErrorLogHandler());
```

Comment:
all the log channels in the app are named "elabftw", I think we should stick to that unless you have a good reason to use a different channel name here.